src/semgrep_agent/semgrep.py

```diff
@@ -126,12 +126,15 @@ def get_findings(
         rewrite_args = [] if rewrite_rule_ids else ["--no-rewrite-rule-ids"]
         metrics_args = ["--enable-metrics"] if enable_metrics else []
         debug_echo("=== seeing if there are any findings")
-        findings = FindingSets()
+        findings = FindingSets(searched_paths=set(targets.searched_paths))
 
         with targets.current_paths() as paths:
             click.echo(
                 "=== looking for current issues in " + unit_len(paths, "file"), err=True
             )
+            if len(paths) < 100:
```

Comment:
Suggestion: remove the length restriction here

Otherwise it's hard to tell what's going on; it should be fine to spew 10k lines of debug log